cypress/integration/handle_administration_link_visibility.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Handle administration link visibility
+    
+    Scenario: A user visits the page
+        Given the user has authority ALL or F_INDICATOR_PUBLIC_ADD
+        When the user accesses any page
+        Then the administration link in the menu is visible
+
+    Scenario: A user visits the page
+        Given the user has authority ALL
+        When the user accesses any page
+        Then the administration link in the menu is visible
+
+    Scenario: A user visits the page
+        Given the user has authority F_INDICATOR_PUBLIC_ADD
+        When the user accesses any page
+        Then the administration link in the menu is visible
```

Comment:
Could you just update the Scenario titles here so they are all slightly different? Otherwise it's hard to see what actually fails if something were to fail

Reply:
fixed